tfe/resource_tfe_team_member.go

```diff
@@ -14,6 +14,9 @@ func resourceTFETeamMember() *schema.Resource {
 		Create: resourceTFETeamMemberCreate,
 		Read:   resourceTFETeamMemberRead,
 		Delete: resourceTFETeamMemberDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
```

Comment:
`resourceTFETeamMemberRead` doesn't save anything:

```
resource "tfe_team_member" "pthrasher" {
  team_id  = "${tfe_team.owners.id}"
  username = "pthrasher"
}
```

ex: `tf import tfe_team_member.pthrasher "team-47qC3LmA47piVWn7|pthrasher"`

```
-/+ tfe_team_member.pthrasher (new resource required)
      id:                                     "team-47qC3LmA47piVWn7|pthrasher" => <computed> (forces new resource)
      team_id:                                "" => "${tfe_team.owners.id}" (forces new resource)
      username:                               "" => "pthrasher" (forces new resource)
```

And shouldn't `team_id` be `team` instead? Or, perhaps you're using `_id` to match the `external_id` and no id means you're using the name?

Reply:
No, it should be the ID. The reasoning is (inline with most/all other TF providers) that if you can create the resource with TF, you use the ID that that resource is using/creating.

I update and added some checks, so the format now also uses a `/` instead of a `|`. And the resource now also sets the `team_id` and `name`.